lib/ansible/config/base.yml

```diff
@@ -685,6 +685,61 @@ DEFAULT_INTERNAL_POLL_INTERVAL:
       Higher values are more suitable for Ansible usage in automation scenarios,
       when UI responsiveness is not required but CPU usage might be a concern.
     - "The default corresponds to the value hardcoded in Ansible <= 2.1"
+DEFAULT_INTERPRETER_PYTHON:
```

Comment:
It's been discussed that we should not use the `DEFAULT_` prefix on new config values. They were kept for older historical "constants", but shouldn't be used going forward.

Reply:
I'd probably just match the env name.